core/src/main/java/org/apache/iceberg/AllEntriesTable.java

```diff
@@ -88,34 +83,16 @@ protected TableScan newRefinedScan(TableOperations ops, Table table, Schema sche
     }
 
     @Override
-    protected String tableType() {
-      return MetadataTableType.ALL_ENTRIES.name();
-    }
+    protected CloseableIterable<FileScanTask> doPlanFiles() {
+      CloseableIterable<ManifestFile> manifests = allManifests(Snapshot::allManifests);
 
-    @Override
-    protected CloseableIterable<FileScanTask> planFiles(
-        TableOperations ops, Snapshot snapshot, Expression rowFilter,
-        boolean ignoreResiduals, boolean caseSensitive, boolean colStats) {
-      CloseableIterable<ManifestFile> manifests = allManifestFiles(
-          ops.current().snapshots(), context().planExecutor());
       String schemaString = SchemaParser.toJson(schema());
       String specString = PartitionSpecParser.toJson(PartitionSpec.unpartitioned());
-      Expression filter = ignoreResiduals ? Expressions.alwaysTrue() : rowFilter;
+      Expression filter = shouldIgnoreResiduals() ? Expressions.alwaysTrue() : filter();
```

Comment:
I think it may be something related to having `ignoreResiduals()` that acts like a setter. I'll have to check the existing implementation. I used what was already available.

Reply:
Yea I see, never mind if its too big a change.

Reply:
Yeah, it is because of a conflicting setter method.

```
@Override
public TableScan ignoreResiduals() {
  return newRefinedScan(ops, table, schema, context.ignoreResiduals(true));
}
```